client/src/screens/search-groups.screen.js

```diff
@@ -94,7 +81,7 @@ class Group extends Component {
       memberAlready: _.some(this.props.myGroups, g => g.id === this.props.group.id),
     };
     return (
-      <TouchableHighlight key={id} onPress={this.joinGroup}>
+      <TouchableHighlight key={id} onPress={() => this.goToGroup(id)}>
```

Comment:
don't pass the `id` to the function here. Use what you had previously (get the group directly from `this.props.group.id`).

Reply:
done